### DISCUSSION TITLE
When there is nothing to do, what you do sometimes?

### DISCUSSION THREAD

**Discussion by iosonoagenda** [community/community#51541] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Just to understand how you get your time productive and useful!

**Comment by CodeMaster7000** [comment, marked as answer]
Hi @iosonoagenda,

You could consider solving issues in repositories of certain organisations by submitting PRs. You could also create issues where you find bugs yourself. Alternatively, you could answer community discussions like this one.

P.S. If you found this answer useful, could you please mark it as an answer? I am trying to get the Galaxy Brain bronze achievement.
